src/scripts/itemDiscuss/dimItemDiscuss.directive.template.html

```diff
@@ -0,0 +1,100 @@
+<div id="discuss-drawer" ng-if="vm.show">
+    <div class="discuss-options">
+        <button class="dim-button" style="float: right;" ng-click="vm.cancel()" translate="Loadouts.Cancel"></button>
+    </div>
+
+    <div class="user-review-form">
+        <div class="user-review-community-overview">
+            <div ng-if="vm.item.communityReviews.length">
+                {{ vm.item.name }} {{ 'DtrReview.HasAverage' | translate }} {{ vm.item.dtrRating }} {{ 'DtrReview.Based' | translate }} {{ vm.item.communityReviews.length }} {{ 'DtrReview.ReviewCount' | translate }}
```

Comment:
Let's avoid string concatenation going forward so that grammar can be addressed correctly.
This should be it own i18n string such as
`{{itemname}} has an average rating of {{itemrating}} based on {{numReviews}} reviews.`
using translate-values="{itemname: vm.item.name, itemrating: vm.item.dtrRating, numReviews: vm.item.communityReviews.length}"